jbmc/src/java_bytecode/assignments_from_json.cpp

```diff
@@ -0,0 +1,828 @@
+/*******************************************************************\
```

Comment:
I don't like `get_value` because the key is called `value` for most types but `@items` for arrays (that's a json-io thing so I didn't want to change it), and the name also doesn't specify _which_ value we're getting (the object without an `@type`). So I'll leave the terminology as it is, but with the definitions in the header file it should hopefully be clearer now.